### PR TITLE
Bug fix 2938

### PR DESCRIPTION
**Pull request by madhuchavva**
### Features and Changes

- Add FallbackAttribute field to the Override rule. 
- Reported Bug: https://github.com/growthbook/growthbook/issues/2938

- Closes **(add link to issue here)**

### Screenshots

image

### CODE REVIEW

**Review by Kevin-Chant — CHANGES_REQUESTED** [review]
The actual change looks simple, but I think you'll need to fix up the diff to the unrelated files first

File: docker-compose.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ version: "3"
 services:
   mongo:
     image: "mongo:latest"
+    ports:
+      - "27017:27017"
```
[the remark this answers is not in the document]
Comment (madhuchavva) [reply docker-compose.yml:6]: Makes it easier for anyone trying to access local mongo docker instance.

File: packages/shared/test/util/features.test.ts [hunk 2]
```diff
@@ -1286,7 +1286,7 @@ describe("validateCondition", () => {
     expect(validateCondition("{(+")).toEqual({
       success: false,
       empty: false,
-      error: "Expected property name or '}' in JSON at position 1",
+      error: "Unexpected token ( in JSON at position 1",
```
[the remark this answers is not in the document]
Comment (madhuchavva) [reply packages/shared/test/util/features.test.ts:1289]: Thought so. Wanted to check with Bryce if these are needed at all.

**Comment by madhuchavva** [comment]
Removed fallback attribute from percentage rollout and force value rules as the sticky bucketing is not supported.

File: packages/shared/test/util/features.test.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (madhuchavva) [reply packages/shared/test/util/features.test.ts]: Reverted.

**Review by Kevin-Chant — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (madhuchavva) [reply docker-compose.yml]: reverted.

**Review by bryce-fitzsimons — APPROVED** [review]

**Pull request merged**